frontend/public/components/operator-lifecycle-manager/_operator-lifecycle-manager.scss

```diff
@@ -104,7 +104,8 @@
   align-items: center;
 }
 
-.co-package-row__actions {
+.co-package-row__actions,
+.pf-c-table.pf-c-virtualized .co-package-row__actions {
```

Comment:
Do we need this for extra specificity or override another style? Would be nice to keep the rules flat as before if possible.

Reply:
this is an old artifact of previously converting the package-manifest / OLM table. I've removed that from this PR and would like to do that conversion as part of the detail tables conversion instead...